Makefile

```diff
@@ -265,4 +265,19 @@ test.clean:
 travis.codecov:
 	$(Q)$(PY_ENV_BIN)/python -m pip install codecov
 
-.PHONY: $(PHONY)
+
+# user-settings
+# -------------
+
+PHONY += user-settings.create user-settings.update
+
+user-settings.update:  pyenvinstall
+	$(Q)$(PY_ENV_ACT); pip install ruamel.yaml
+	$(Q)$(PY_ENV_ACT); python utils/update_user_settings.py ${SEARX_SETTINGS_PATH}
+
+user-settings.update.engines:  pyenvinstall
+	$(Q)$(PY_ENV_ACT); pip install ruamel.yaml
```

Comment:
Does it make sense to add `ruamel.yaml` to `requirements-dev.txt`?

Reply:
The purpose is not to develop / test code, but to update the configuration.

`requirements.txt` fits better, so it would make sense to:
* remove the `pyyaml` dependency
* rely only on `ruamel.yaml`

But it is only requires in this specific use case, so I went for this hybrid approach.